project.godot

```diff
@@ -30,9 +30,9 @@ config/windows_native_icon="res://assets/misc/icon.ico"
 
 [autoload]
 
-StartupActions="*res://src/engine/StartupActions.cs"
-SimulationParameters="*res://simulation_parameters/SimulationParameters.cs"
 Invoke="*res://src/engine/Invoke.cs"
+SimulationParameters="*res://simulation_parameters/SimulationParameters.cs"
+StartupActions="*res://src/engine/StartupActions.cs"
```

Comment:
Unless there's a reason to change the order of these two. They should be kept in the same relative order, so StartupActions should be before simulation parameters like it was before.

Reply:
At the end of ApplyLanguageSettings() in Settings.cs, we have SimulationParameters.Instance.ApplyTranslations() require SimulationParameters to load before or the game crash. If I understand, ApplyLanguageSettings need to be called directly and not with Invoke, so I don't have the choice to move that ?

Reply:
That's probably correct. It's fine to leave as is.
I initially decided to put startup actions before simulation parameters in order to allow something like a mod loader to modify simulation parameter loading options.